docs/changelog.md

```diff
@@ -18,9 +18,29 @@ encapsulated this logic;
 - Constructors for ScMemoryContext with string and int parameters were removed. Use the one without parameters instead.
 - HelperSearchTemplateInStruct was removed from ScMemoryContext API. Use HelperSearchTemplate with callbacks.
 - Deprecated sc-utils in 0.9.0 were removed from sc-kpm.
+- We've made breakthrough. Now we don't use code generation as metaprogramming. The API for agents has been completely redesigned. See the documentation section on how to implement an agent with the new API. What have we done?
+  - Firstly, we have completely moved from code generation to template programming.
+  - Secondly, we have significantly improved and aligned with our description of how it should be.
+  - Thirdly, we have significantly simplified the API. Now it will be much easier to create agents. Go right now and see how to work with the new API -- [C++ Agents API](sc-memory/api/cpp/extended/agents/agents.md).
 
 ### Added
 
+- Simple guide for implementing agent in C++
```

Comment:
Users will rate how easy this guide is

Reply:
Here `simple` means that the guide is not easy to read, but that it minimally shows how to implement the agent

Reply:
Maybe, I should to rename it to `Minimum` guide?